diff --git a/soh/src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c b/soh/src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c
index 715ca55..f68fedb 100644
--- a/soh/src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c
+++ b/soh/src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c
@@ -138,6 +138,11 @@ void EnBomChu_UpdateFloorPoly(EnBomChu* this, CollisionPoly* floorPoly, GlobalCo
     f32 normDotUp;
     MtxF mf;
 
+    if (floorPoly == NULL) {
+        EnBomChu_Explode(this, globalCtx);
+        return;
+    }
+
     this->actor.floorPoly = floorPoly;
 
     normal.x = COLPOLY_GET_NORMAL(floorPoly->normal.x);
